src/xenia/app/xenia_main.cc

```diff
@@ -67,7 +67,7 @@ DEFINE_path(
     "Storage");
 
 DEFINE_bool(mount_scratch, false, "Enable scratch mount", "Storage");
-DEFINE_bool(mount_cache, false, "Enable cache mount", "Storage");
+DEFINE_bool(mount_cache, true, "Enable cache mount", "Storage");
```

Comment:
This one is kinda ugh because it won't be changed for existing users since they already have the entry in the config :/

Reply:
Do you suggest changing the variable name? imho it is a rather good name right now

Reply:
I wonder if it's worth having a versioning system for cvars? would be a pain to implement though

Reply:
Thought about something similar — something like `DEFAULT_CHANGED_bool(mount_cache, false, 2020, 10, 26, 13);` with the constructor `max`ing the global current defaults version, and a config variable containing the defaults version at the moment of config writing. Hacky, but possibly more user- and parsing-friendly (as opposed to comments, which would be pain to treat as special defaults comments for sorting purpose), and won't need to wait for the majority of users to upgrade their config to the comments+overrides format before changing defaults safely.

Reply:
The version would be (in UTC+0 time) `hour + day * 100 + month * 10000 + year * 1000000` — fits in int32.

Reply:
why not simply comment out default cvars in the config on writing? If the default has changed on the next load it will automatically apply.

Reply:
Maybe we should create issue for that and continue there as it isn't exactly related to this PR 🤷

Reply:
@JoelLinn Oh, yes, since we're overwriting the entire file deleting all user-added comments, it should work fine without sorting issues. There are two issues though:
* Users will have to be told to remove # before the line whenever they change something — we're going to have a lot of "I changed this and it doesn't work" questions.
* If a user, for instance, has last used Xenia in September, we add the comment system in October, change some default in November, then a user updating Xenia in December will not get the new default.

Reply:
@Triang3l That's actually easier than you might think.

We could add a new general config version cvar, which Xenia checks on startup, if it's not present or has a value less than we expect, we can prompt the user that their config file may have issues due to being out of date.

I can agree with commenting out values that match the default.

Reply:
That's what I had in mind, though instead of some centralized version number location, per-cvar history, and automatic upgrading if the value matches a previous default, rather than a warning that will make users have to drop their entire config.

Reply:
I'd be down for that as well, would just need to be in a form that's user-friendly.

Reply:
The needed functionality has been added in 37013ee352da93e8e2374bd252d3c8a27ba71a1a — after defining `mount_cache`, add:
`UPDATE_from_bool(mount_cache, year, month, day, hour, false);`
and update the `IConfigVarUpdate::kLastCommittedUpdateDate` to the same date in `base/cvar.h`.